ts/Accessibility/Components/RangeSelectorComponent.ts

```diff
@@ -690,6 +690,7 @@ namespace RangeSelectorComponent {
      */
     export function compose(
         ChartClass: typeof Chart,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
As above, this one looks like a mistake. Why pass RangeSelectorClass when it's not used?